docs/index.md

```diff
@@ -58,36 +59,36 @@ This short clip briefly demonstrates containerlab features and explains its purp
-* **Multi-vendor and open**  
+* **Multi-vendor and open**
     Although being kick-started by Nokia engineers, containerlab doesn't take sides and supports NOSes from other vendors and opensource projects.
-* **Lab orchestration**  
+* **Lab orchestration**
     Starting the containers and interconnecting them alone is already good, but containerlab packages even more features like managing lab lifecycle: [deploy](cmd/deploy.md), [destroy](cmd/destroy.md), [save](cmd/save.md), [inspect](cmd/inspect.md), [graph](cmd/graph.md) operations.
-* **Scaled labs generator**  
+* **Scaled labs generator**
     With [`generate`](cmd/generate.md) capabilities of containerlab it possible to define/launch CLOS-based topologies of arbitrary scale. Just say how many tiers you need and how big each tier is, the rest will be done in a split second.
-* **Simplicity and convenience**  
+* **Simplicity and convenience**
     Starting from frictionless [installation](install.md) and [upgrade](install.md#upgrade) capabilities and ranging to the behind-the-scenes [link wiring machinery](manual/network.md), containerlab does its best for you to enjoy the tool.
-* **Fast**  
+* **Fast**
     Blazing fast way to create container based labs on any Linux system with Docker.
-* **Automated TLS certificates provisioning**  
+* **Automated TLS certificates provisioning**
     The nodes which require TLS certs will get them automatically on boot.
-* **Documentation is a first-class citizen**  
+* **Documentation is a first-class citizen**
     We do not let our users guess by making a complete, concise and clean [documentation](https://containerlab.dev).
-* **Lab catalog**  
+* **Lab catalog**
    The "most-wanted" lab topologies are [documented and included](lab-examples/lab-examples.md) with containerlab installation. Based on this cherry-picked selection you can start crafting the labs answering your needs.
 
 ## Use cases
 
-* **Labs and demos**  
-    Containerlab was meant to be a tool for provisioning networking labs built with containers. It is free, open and ubiquitous. No software apart from Docker is required!  
-    As with any lab environment it allows the users to validate features, topologies, perform interop testing, datapath testing, etc.  
+* **Labs and demos**
+    Containerlab was meant to be a tool for provisioning networking labs built with containers. It is free, open and ubiquitous. No software apart from Docker is required!
+    As with any lab environment it allows the users to validate features, topologies, perform interop testing, datapath testing, etc.
     It is also a perfect companion for your next demo. Deploy the lab fast, with all its configuration stored as a code -> destroy when done. Easily and [securely share lab access](manual/published-ports.md) if needed.
-* **Testing and CI**  
```

Comment:
your formatter removed trailing spaces here, but they are needed

Reply:
Hi, I did not realised that my VScode formatter removed them as it was rendered correctly by `mkdocs`. I know now ! Thanks for the corrections !